src/ui/Assets/Styles/site.scss

```diff
@@ -106,3 +107,7 @@ $success-green: #00823b;
   @include govuk-font($size: 16, $line-height: 1.5);
   color: $govuk-text-colour;
 }
+
+.pointer-events-none {
```

Comment:
This relates to this bug:
https://github.com/alphagov/govuk-frontend/issues/1252